README.md

```diff
@@ -1,2 +1,37 @@
-# cookiecutter_ffrd_template
-Cookiecutter template for generating a new toolkit repository
+# cookiecutter-ffrd-template
+Cookiecutter template for generating a new toolkit repository.
+
+Features
+- MIT License  
+- Dev Container support  
+- A functioning testing template as guidance  
+- Other version control best practices built-in
+
+
+### Requirements to use the cookiecutter template:
+-----------
+ - Python 2.7 or 3.5+
+ - [Cookiecutter Python package](http://cookiecutter.readthedocs.org/en/latest/installation.html) >= 1.4.0: This can be installed with pip by or conda depending on how you manage your Python packages:
+
+Pip:  
+``` bash
+$ pip install cookiecutter
+```
+
+Conda [or micromamba]:
+``` bash
+$ conda [micromamba] install -c conda-forge cookiecutter
+```
+
+
+### To start a new project, run:
```

Comment:
Remove _", run"_ from line 27 - it's basically understood.

- [ ] Remove "run"